driver-tests/osgi/src/test/java/com/datastax/driver/osgi/BundleOptions.java

```diff
@@ -35,19 +35,19 @@ public static UrlProvisionOption driverBundle() {
 
     public static UrlProvisionOption driverBundle(boolean useShaded) {
         String classifier = useShaded ? "-shaded" : "";
-        return bundle("reference:file:" + PathUtils.getBaseDir() + "/../../driver-core/target/cassandra-driver-core-" + projectVersion() + classifier + ".jar");
+        return bundle("reference:file:" + PathUtils.getBaseDir() + "/../../driver-core/target/cassandra-driver-core-" + Cluster.getDriverVersion() + classifier + ".jar");
```

Comment:
Nice, hadn't thought of at the time to use`getDriverVersion()`, definitely is much more elegant.